templates/partials/usages.html

```diff
@@ -0,0 +1,34 @@
+<h2>What can you build?</h2>
+<dl class="features">
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="gears") }}
+        GUI Applications
+      </div>
+    </dt>
+    <dd>Perfect integration of GTK, Libadwaita, Libgranite makes development super simple. Publish easily your apps on Flathub without bloated packages.</dd>
```

Comment:
The Flathub part is more of a platform thing that would make more sense in https://developer.gnome.org.

(In fact there's a dedicated section and page for Flatpak there :sweat_smile:).

Reply:
```suggestion
    <dd>Perfect integration of GTK, Libadwaita and Libgranite makes development super simple. Publish easily your apps on Flathub without bloated packages.</dd>
```

Reply:
> The Flathub part is more of a platform thing that would make more sense in https://developer.gnome.org.
> 
> (In fact there's a dedicated section and page for Flatpak there sweat_smile).

For GUI apps flathub is the standard and everywhere used. And Vala makes actually publishing really easier. Other bindings are more difficult to setup.